### PR TITLE
chore(css-prefix): add animation

### PR DESCRIPTION
**Pull request by magsout**
## Summary

Part of #1423.

Adds `animation` related prefixes

## Test Plan

ci

### CODE REVIEW

File: internal/compiler/transforms/compile/css-handler/prefix/prefixes/animation.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"animation",
+	"animation-name",
+	"animation-duration",
+	"animation-delay",
+	"animation-direction",
+	"animation-fill-mode",
+	"animation-iteration-count",
+	"animation-play-state",
+	"animation-timing-function",
+	"@keyframes",
```
Comment (jer3m01) [inline internal/compiler/transforms/compile/css-handler/prefix/prefixes/animation.ts:18]: You can remove `@keyframes` they'll be in a separate special file as `prefixCSSProperty` can't handle them.

**Review by jer3m01 — CHANGES_REQUESTED** [review]
Else than my comment everything looks good!

File: internal/compiler/transforms/compile/css-handler/prefix/prefixes/animation.ts (hunk 1 above)
Comment (magsout) [reply internal/compiler/transforms/compile/css-handler/prefix/prefixes/animation.ts]: done

**Comment by jer3m01** [comment]
Thanks!

**Pull request merged**